src/pages/my-patches/patch-card/BuildStatusIcon/CreatedIcon.ts

```diff
@@ -0,0 +1,9 @@
+import { uiColors } from "@leafygreen-ui/palette";
+import styled from "@emotion/styled";
+
+export const CreatedIcon = styled.div`
+  border-radius: 50%;
+  border: 2px solid ${uiColors.gray.light1};
```

Comment:
can we destructure this.

Reply:
if `uiColors` is used once in this file then what is the value of destructuring this?

Reply:
I think i left the comment as a force of habit from years of always having the [prefer-destructuring](https://eslint.org/docs/rules/prefer-destructuring) rule enforced. Its fine if we dont use it.